src/components/Atoms/SVGAtom/SkillTagAtom/SkillTagAtom.tsx

```diff
@@ -0,0 +1,18 @@
+import React from 'react'
+import SVGAtom from '../../../Atoms/SVGAtom/SVGAtom'
+
+type Props = {
+  name: string;
+  level: number;
+  icon: "html5" | "bootstrap" | "search";
```

Comment:
if you use typeof SVGAtom's iconName, use `React.ComponentProps<typeof SVGAtom>['iconName'];`